docker-compose.yaml

```diff
@@ -7,7 +7,7 @@ services:
     environment:
       - "PROVIDER_BASE_URLS=https://meshery.layer5.io"
       - "SAAS_BASE_URL=https://meshery.layer5.io"
-      - "ADAPTER_URLS=meshery-istio:10000 meshery-linkerd:10001 meshery-consul:10002 meshery-nsm:10004 meshery-app-mesh:10005 meshery-kuma:10007 meshery-cpx:10008 meshery-osm:10009 meshery-traefik-mesh:10006 meshery-nginx-sm:10010"
+      - "ADAPTER_URLS=meshery-istio:10000 meshery-linkerd:10001 meshery-consul:10002 meshery-nsm:10004 meshery-app-mesh:10005 meshery-kuma:10007 meshery-cpx:10008 meshery-osm:10009 meshery-traefik-mesh:10006 meshery-nginx-sm:10010 meshery-cilium:10011"
```

Comment:
Thanks for this, @piyushsingariya. It's actually 10012. Have a look at the docs.meshery.io (a port is reserved there).

Reply:
```suggestion
      - "ADAPTER_URLS=meshery-istio:10000 meshery-linkerd:10001 meshery-consul:10002 meshery-nsm:10004 meshery-app-mesh:10005 meshery-kuma:10007 meshery-cpx:10008 meshery-osm:10009 meshery-traefik-mesh:10006 meshery-nginx-sm:10010 meshery-cilium:10012"
```